Source/Fuse.Maps/MapTile.uno

```diff
@@ -0,0 +1,298 @@
+		Fuse.Controls.Image[] maps = new Fuse.Controls.Image[9];
+		Fuse.Controls.Grid grid;
+		MarkerIconCache _markerGraphicsCache;
+
+		public MapTile()
+		{
+			grid = new Fuse.Controls.Grid()
+			{
+				ColumnCount = 3
+			};
+			var i = 0;
+			for (var y = 1; y < 4; y++)
+			{
+				for (var x = 1; x < 4; x++)
+				{
+					maps[i] = new Fuse.Controls.Image()
+					{
+						Url = MakeUrl(2, x, y)
+					};
+					grid.Add(maps[i]);
+					i++;
+				}
+			}
+			trans = new Fuse.Translation();
+			var scaling = new Fuse.Scaling()
+			{
+				Factor=1.5f
+			};
+			Children.Add(grid);
+			Children.Add(trans);
+			Children.Add(scaling);
+			Placed += OnPlaced;
+			_markerGraphicsCache = new MarkerIconCache(UpdateMarkers);
+		}
+
+		// http://wiki.openstreetmap.org/wiki/Zoom_levels
+		// http://wiki.openstreetmap.org/wiki/Slippy_map_tilenames
+		String _tileserver = "http://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png";
+		public String TileServer
+		{
+			get { return _tileserver; }
+			set { _tileserver = value; }
+		}
+
+		public String MakeUrl (int zoom, int x, int y)
+		{
+			var s = TileServer
+						.Replace("{s}", "{0}")
+						.Replace("{z}", "{1}")
+						.Replace("{x}", "{2}")
+						.Replace("{y}", "{3}");
+			if (x < 0) return null;
+			if (y < 0) return null;
+			if (zoom < 0) return null;
+			return String.Format(s, "a", zoom, x, y);
+		}
+
+		public ObservableList<MapMarker> Markers
+		{
+			get
+			{
+				MapView m = Parent as MapView;
+				if (m != null)
+				{
+					return m.Markers;
+				}
+				return null;
+			}
+		}
+
+		public void ClearMarkers ()
+		{
+			foreach(var i in _markers)
+			{
+				Children.Remove(i);
+			}
+			_markers.Clear();
+		}
+
+		List<Fuse.Controls.Image> _markers = new List<Fuse.Controls.Image>();
+		public void UpdateMarkers()
+		{
+			ClearMarkers();
+			foreach(MapMarker m in Markers)
+			{
+				FileSource fs = m.IconFile;
+				if (fs == null)
+				{
+					fs = new global::Uno.UX.BundleFileSource(import global::Uno.IO.BundleFile("./mappin.png"));
+					m.IconAnchorY = 1;
+					m.IconAnchorX = 0;
+				}
+				// The diff between the panel and the grid. Offset this to center the marker
+				var diff = (ActualSize - _grid_size) / 2;
+				var p = WorldToTilePos(m.Longitude, m.Latitude, (int)Zoom);
+
+				// P is now relative to the top corner of the grid
+				p -= _cornertile;
+				// P is now in pixels
+				p = p * (_grid_size / 3);
+				p += diff;
+
+				var i = new Fuse.Controls.Image()
+				{
+					File = fs,
+					Width = 10,
+					X = p.X,
+					Y = p.Y
+				};
+				var _trans = new Fuse.Translation()
+				{
+					RelativeNode = i,
+					RelativeTo   = Fuse.TranslationModes.Size,
+					X = m.IconAnchorX - 0.5f,
+					Y = m.IconAnchorY * -1
+				};
+
+				Children.Add(i);
+				BringToFront(i);
+				i.Children.Add(_trans);
+				_markers.Add(i);
+			}
+		}
+
+		bool ready = false;
+		protected override void OnRooted()
+		{
+			base.OnRooted();
+			MapView m = Parent as MapView;
+			m.ClipToBounds = true;
```

Comment:
This works as I want when I have `ClipToBounds` on the parent, and it doesn't work when I put it on this Panel itself. The reason is that I'm adding a `Translation` on the Panel itself, to move it, so that the wanted center of the map is centered. And then I clip with the parent, to not show the map outside the bounds. This is a workaround, but it was how I made it work.